### PR TITLE
Add. Integrations. Add HubSpot integration, extend logic for external forms handler.

### PR DESCRIPTION
**Pull request by svfcode**

### CODE REVIEW

File: js/src/apbct-public--5--external-forms.js [hunk 1]
```diff
@@ -376,3 +377,99 @@ function sendAjaxCheckingFormData(form, prev, formOriginal) {
...
+        }
+    });
+
+    apbct_public_sendAJAX(
+        data,
+        {
+            async: false,
+            callback: function(result) {
+                if( result.apbct === undefined || ! +result.apbct.blocked ) {
+                    form.onsubmit = null
```
Comment (Glomberg) [inline js/src/apbct-public--5--external-forms.js:460]: Need to store the original submit handler and will return back it here

**Review by Glomberg — CHANGES_REQUESTED** [review]
https://github.com/CleanTalk/wordpress-antispam/pull/229/files#r1099734587

File: js/src/apbct-public--5--external-forms.js (hunk 1 above)
Comment (svfcode) [reply js/src/apbct-public--5--external-forms.js]: Done.

File: js/src/apbct-public--5--external-forms.js [hunk 2]
```diff
@@ -376,3 +378,102 @@ function sendAjaxCheckingFormData(form, prev, formOriginal) {
...
+    let neededFormIds = [];
+    for (let form of forms) {
+        if (form.id.indexOf('hsForm') !== -1) {
+            neededFormIds.push(form.id)
+        }
+    }
+
+    for (let formId of neededFormIds) {
+        let form = documentObject.getElementById(formId);
+        form.onsubmit_prev = form.onsubmit;
```
Comment (Glomberg) [inline js/src/apbct-public--5--external-forms.js:415]: The object `onsubmit_prev` is already defined by visible fields logic. Please, rename this.

**Review by Glomberg — CHANGES_REQUESTED** [review]
https://github.com/CleanTalk/wordpress-antispam/pull/229/files#r1104003613

**Review by ArtemAnoshin — APPROVED** [review]

File: js/src/apbct-public--5--external-forms.js (hunk 2 above)
Comment (Glomberg) [reply js/src/apbct-public--5--external-forms.js]: https://github.com/CleanTalk/wordpress-antispam/pull/229/commits/3d1533010a866b4faa09dee70e0e0fd14384ceb6

**Pull request merged**